src/objects/logo-group/logo-group.stories.mdx

```diff
@@ -0,0 +1,13 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import defaultDemo from './demo/demo.twig';
```

Comment:
There's an opportunity to refine the story code example by using the Webpack `!!raw-loader` syntax here as well. 

Also see [the next suggestion](). 😉 
```suggestion
// The '!!raw-loader!' syntax is a non-standard, Webpack-specific, syntax.
// See: https://github.com/webpack-contrib/raw-loader#examples
// For now, it seems likely Storybook is pretty tied to Webpack, therefore, we are
// okay with the following Webpack-specific raw loader syntax. It's better to leave
// the ESLint rule enabled globally, and only thoughtfully disable as needed (e.g.
// within a Storybook docs page and not within an actual component).
// This can be revisited in the future if Storybook no longer relies on Webpack.
// eslint-disable-next-line @cloudfour/import/no-webpack-loader-syntax
import defaultDemoSource from '!!raw-loader!./demo/demo.twig';
import defaultDemo from './demo/demo.twig';
```